biome.json

```diff
@@ -1,8 +1,5 @@
 {
   "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
-  "organizeImports": {
-    "enabled": false
```

Comment:
Why this change?

Reply:
it was a duplicate, we already have this property below in the same file.

Reply:
Ahh thanks!